allensdk/brain_observatory/behavior/session_apis/data_io/behavior_lims_api.py

```diff
@@ -252,16 +255,23 @@ def get_rig_name(self) -> str:
 
     @memoize
     def get_stimulus_name(self) -> str:
-        """Returns the name of the stimulus set used for the session.
+        """Get the stimulus set used from the behavior session pkl file
         :rtype: str
         """
-        query = f"""
-            SELECT stages.name
-            FROM behavior_sessions bs
-            JOIN stages ON stages.id = bs.state_id
-            WHERE bs.id = '{self.foraging_id}';
-        """
-        return self.mtrain_db.fetchone(query, strict=True)
+        behavior_stimulus_path = self.get_behavior_stimulus_file()
+        pkl = pd.read_pickle(behavior_stimulus_path)
+
+        try:
+            stimulus_name = pkl["items"]["behavior"]["cl_params"]["stage"]
```

Comment:
what's the difference between `params` and `cl_params`? I've seen both and they've been the same in the cases I have seen.